src/components/StoryInput/StoryInput.js

```diff
@@ -18,19 +18,29 @@ const messages = defineMessages({
   }
 })
 
-function StoryInput () {
-  return (
-    <div className={ s.root } >
-      <div className={ s.container } >
-        <textarea
-          rows="3"
-          placeholder={ messages.inputPlaceholder.defaultMessage }
-          className={ cn(su.noOutline, su.noResize) }
-        />
-        <button><FormattedMessage { ...messages.button }  /></button>
+class StoryInput extends Component {
+  render (){
+    return (
+      <div className={ s.root } >
+        <div className={ s.container } >
+          <textarea
+            rows="3"
+            placeholder={ messages.inputPlaceholder.defaultMessage }
```

Comment:
inputPlaceholder is too broad, if we had to add more inputs and following this pattern we would always have to change the inputPlaceholder to something like storyInputPlaceholder.